packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/container.tsx

```diff
@@ -43,7 +45,9 @@ const ResetPasswordWithRecoveryKeyVerifiedContainer = ({
     useState<FinishOAuthFlowHandlerResult['error']>();
 
   const account = useAccount();
-  const accountResetData = sensitiveDataClient.getData('accountResetData');
+  const { uid, sessionToken } = currentAccount() || {};
+  const { keyFetchToken, unwrapBKey } =
+    sensitiveDataClient.getDataType(SensitiveData.Key.AccountReset) || {};
   const newRecoveryKeyData = sensitiveDataClient.getData('newRecoveryKeyData');
```

Comment:
I filed FXA-10849 for moving over the remaining entries so that it doesn't snowball further. I'd like to land this early in the sprint with enough bake time and minimal conflicts. 😄